JSDemos/Demos/DataGrid/UsingHeaderFilter/description.md

```diff
@@ -8,6 +8,6 @@ Assign **true** to the **headerFilter**.[visible](/Documentation/ApiReference/UI
 
 Users can search values within a header filter. To configure the search panel, define the [search](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/headerFilter/search/) property in the global [headerFilter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/headerFilter/) object or in a **columns**.[headerFilter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/headerFilter/) object. 
 
-You can compare the search string against specific fields in your data objects. For this purpose, apply the **search**.[searchExpr](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/headerFilter/search/#searchExpr) property. In this example, the search string is compared against the "City" and "State" fields.
+The search panel's default behavior is to look for values only within the same data field. You can include additional fields into the search. Specify the **search**.[searchExpr](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/headerFilter/search/#searchExpr) property. For example, this demo allows you to enter a state name into the City column's header filter. You can then see a list of all cities within the specified state, and select city names that you want to use as a filter. 
```

Comment:
The search panel's default behavior looks for values only within the same data field. You can include additional fields in the search. Specify the **search**.[searchExpr] property. For example, this demo allows you to enter a state name in the City column's header filter. You can then see a list of all cities within the specified state, and select city names that you want to use as a filter.